server/server/src/lib.rs

```diff
@@ -57,11 +57,16 @@ macro_rules! internal {
 }
 
 pub fn verify_client_version<Req: Request>(
```

Comment:
```suggestion
pub fn handle_version<Req: Request>(
```

to suggest that this fn has a side effect